src/app/query/bitcoin/runes/runes.hooks.ts

```diff
@@ -10,8 +14,19 @@ function makeRuneToken(rune: RuneBalance): RuneToken {
   };
 }
 
+export function useRunesEnabled() {
+  const runesEnabled = useConfigRunesEnabled();
+  const network = useCurrentNetwork();
+
+  return runesEnabled && network.chain.bitcoin.bitcoinNetwork === 'testnet';
```

Comment:
Here too? This would only work on testnet?

Reply:
I was using this to determine if live on mainnet, so if naming is off we can change it.